pimd/pim_bsm.c

```diff
@@ -1362,8 +1352,8 @@ int pim_bsm_process(struct interface *ifp, pim_sgaddr *sg, uint8_t *buf,
 		} else {
 			if (PIM_DEBUG_BSM)
 				zlog_debug(
-					"%s : nofwd_bsm received on %s when accpt_nofwd_bsm false",
-					__func__, bsr_str);
+					"%s : nofwd_bsm received on %pPAs when accpt_nofwd_bsm false",
+					__func__, &bshdr->bsr_addr.addr);
```

Comment:
Need a `(pim_addr *)` cast here since the extended format checking in CI will print a warning for `%pPA` with `struct in_addr *` / `struct in6_addr *`

Reply:
Done